Code/Editor/CryEdit.cpp

```diff
@@ -3959,32 +3960,64 @@ void CCryEditApp::OpenLUAEditor(const char* files)
         {
             for (const auto& resolvedPath : resolvedPaths)
             {
-                args.append(AZStd::string::format(" -files %s", resolvedPath.c_str()));
+                args.append(AZStd::string::format("\"-files=%s\"", resolvedPath.c_str()));
```

Comment:
Can quote just the files value, not the whole argument.
```suggestion
                args.append(AZStd::string::format(R"(-files="%s")", resolvedPath.c_str()));
```